libraries/AP_InertialSensor/AP_InertialSensor_tempcal.cpp

```diff
@@ -68,7 +68,7 @@ const AP_Param::GroupInfo AP_InertialSensor::TCal::var_info[] = {
     // @Units: degC
     // @User: Advanced
     // @Calibration: 1
-    AP_GROUPINFO("TMAX", 3, AP_InertialSensor::TCal, temp_max,  0),
+    AP_GROUPINFO("TMAX", 3, AP_InertialSensor::TCal, temp_max,  70),
```

Comment:
I just did a calibration last night and my baro temp went up to 73C

Reply:
yes, that's fine, you can set it anything you want. I just made it 70 as Henry misread the docs and didn't know it needed to be set, so his cal stopped at 0C